02-peer-protocol.md

```diff
@@ -487,6 +487,9 @@ so we should fulfill the incoming HTLC as soon as we can to reduce latency.
 ### Risks With HTLC Timeouts
 
 
+Once an HTLC has timed out where it could either be fulfilled or timed-out;
```

Comment:
I don't understand the first part of the sentence, but that's probably because my english is not good enough.